src/ServicePulse.Host/vue/src/stores/CustomChecksStore.ts

```diff
@@ -0,0 +1,53 @@
+import { useDeleteFromServiceControl, useTypedFetchFromServiceControl } from "@/composables/serviceServiceControlUrls";
+import CustomCheck from "@/resources/CustomCheck";
+import { acceptHMRUpdate, defineStore } from "pinia";
+import { ref, watch } from "vue";
+import useAutoRefresh from "@/composables/autoRefresh";
+
+export const useCustomChecksStore = defineStore("CustomChecksStore", () => {
+  const prefix = "customchecks/";
+
+  const pageNumber = ref(1);
+  const failingCount = ref(0);
```

Comment:
could be a calculated value of failedChecks.length

Reply:
It isn't that though. `failedChecks` is a page of data. `failingCount` is the total, irrespective of paging. It is used to figure out the number of pages.